cypress/integration/errorPageArticle.js

```diff
@@ -0,0 +1,73 @@
+import {
+  errorMessage,
+  errorPageInlineLink,
+  errorTitle,
+  hasHtmlLangDirAttributes,
+} from '../support/bodyTestHelper';
+import testData from '../../src/app/lib/config/services';
+import services from '../support/config/services';
+
+const nonExistentAsset = 'cxvxrj8tvppo';
```

Comment:
This should never be done, the value should be taken from the support/config/services.js file

Reply:
Sorry this definitely needs removing ...added it during debugging process